### PR TITLE
Tool to export CF host images to GCE.

### PR DESCRIPTION
**Pull request by ser-io**

### CODE REVIEW

File: scripts/gcp/create_host_image.sh [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    \?)
+      echo "Invalid option: ${OPTARG}" >&2
+      usage
+      exit 1
+      ;;
+    :)
+      echo "Invalid option: ${OPTARG} requires an argument" >&2
+      usage
+      exit 1
+      ;;
```
Comment (jemoreira) [inline scripts/gcp/create_host_image.sh:63]: nit: These two could be combined into
```
    \?)
      usage; exit 1
```
If you remove the starting `:` from the option string. The default messages printed by `getopts` to stderr are very similar to yours.

**Review by jemoreira — COMMENTED** [review]
LGTM but the script is quite complex, especially the curl in a loop with json parsing. Have you considered writing this in go like other scripts we have in aosp?

File: scripts/gcp/create_host_image.sh (hunk 1 above)
Comment (ser-io) [reply scripts/gcp/create_host_image.sh]: Done.

**Comment by ser-io** [comment]
> LGTM but the script is quite complex, especially the curl in a loop with json parsing. Have you considered writing this in go like other scripts we have in aosp?

yes, the pagination logic due the lack of a search endpoint in the Github Artifacts API does not look appealing in bash. However the script story is simple: "search token" -> "download" -> "upload" which I still think is a good fit for bash.

**Review by jemoreira — APPROVED** [review]

**Pull request merged**